### PR TITLE
updated file

### PR DESCRIPTION
**Pull request by akaevjumgal**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1 @@
-hello world I updated this file GET OUT!
+hello world
```
Comment (Bonecide) [inline test.txt]: its me

Comment (fAyday-everyday) [inline test.txt]: asdasd

File: test.txt [hunk 2]
```diff
@@ -1 +1 @@
-hello world I updated this file GET OUT!
```
Comment (ranbu171) [inline test.txt:1]: Hi :)

**Comment by tmukaev** [comment]
IJNbhuqQ

File: test.txt (hunk 1 above)
Comment (akaevjumgal) [reply test.txt]: I fix it

**Pull request merged**